### ISSUE TITLE
Retab reopens tabs opened at incognito mode

### ISSUE THREAD

**Issue by taraslayshchuk** [bucaran/retab#24]
With shortcut I can get any tab which was opened at incognito mode.

**Comment by bucaran** [comment]
If I recall correctly, retab remembers incognito tabs. Is it broken now?

**Comment by taraslayshchuk** [comment]
So it was designed? I think it is not good to have history of incognito tabs or best way to have way to disable this.

**Comment by bucaran** [comment]
@taraslayshchuk Ok, my mistake, it's **disabled** during incognito mode. 

I looked at the code again (it's been almost 2 years) and you can see here:

* https://github.com/bucaran/retab/blob/master/retab.safariextension/index.js#L22 

that during incognito mode we don't record tabs, which is the desired behavior right?

**Comment by taraslayshchuk** [comment]
Yeah, but at my web browser I can open tabs from incognito. How to reproduce: 
- open safari and safari in incognito mode
- open a few tabs in incognito and close them all
- close a incognito mode
- push shortcut in normal mode for execute retab(cmd+shift+tab) and you will have in normal mode tabs which was opened and closed at incognito

**Comment by bucaran** [comment]
Ha! I see, it seems that you found a bug :wink:

**Comment by taraslayshchuk** [comment]
Exactly! ;]

**Comment by scarfacedeb** [comment]
any news on this one?
